Flutter/linear-gauge/animation.md

```diff
@@ -33,7 +33,7 @@ The [`animateAxis`](https://pub.dev/documentation/syncfusion_flutter_gauges/late
 
 {% endhighlight %}
 
-![Animate axis in linear gauge](images/animation/animation_axis_range/animation_axis.gif)
+![Animate axis in linear gauge](images/animation/animation-axis-range/animation_axis.gif)
```

Comment:
Don't use underscore in the file name too

Reply:
Yes, I have changed to hyphen (-).